android/app/src/main/java/app/covidshield/module/ExposureNotificationModule.kt

```diff
@@ -121,6 +122,33 @@ class ExposureNotificationModule(context: ReactApplicationContext) : ReactContex
         }
     }
 
+    @ReactMethod
+    fun getPendingExposureSummary(promise: Promise) {
+        promise.launch(this) {
+            val tokens = PendingTokenManager.instance.retrieve()
+            PendingTokenManager.instance.clear()
+
+            val summaryAndToken = tokens
```

Comment:
If there is more than one token is pending, we could assume that one token has matchedKeyCount is enough to say people is exposed.